ext/ox/builder.c

```diff
@@ -398,6 +400,7 @@ builder_new(int argc, VALUE *argv, VALUE self) {
  * - +options+ - (Hash) formating options
  *   - +:indent+ (Fixnum) indentaion level, negative values excludes terminating newline
  *   - +:size+ (Fixnum) the initial size of the string buffer
+ *   - +:effort+ [:strict|:tolerant|:auto_define] set the tolerance level for writing text
```

Comment:
The :effort options is not supported by the Builder. If the intent is to strip invalids then that option should be specified as a boolean instead of reusing :effort.

Reply:
Indeed, I removed it but forgot to update doc.